### PR TITLE
Adds portable ammo packets for the SH15!

### PR DESCRIPTION
**Pull request by Mrrpip**
## About The Pull Request

https://user-images.githubusercontent.com/38842059/238773907-f17597f9-7d36-4c71-b77e-18a0a012e4b6.PNG

Yeah. It only took 3 or so months for me to bother again (Stupid merge conflicts.). Anyhow, adds two ammo boxes for the SH-15, for their corresponding ammo types.

It also changes the Slug magazine to be green. Because I didn't want to make a red box to then change it to green,

## Why It's Good For The Game

I think it'd be more enjoyable while not affecting balance to a super duper degree to reward recycling mags...Or something. Explaining stuff isn't my forte.

## Changelog
:cl:
add: 16G ammo boxes have arrived, now you'll get to use more shotgun in your autoshotgun!
imageadd: The SH-15 slug mag is now green. 
/:cl:

### CODE REVIEW

**Comment by Mrrpip** [comment]
For reference, it contains 60 ammo or 4 mags as of the posting of the PR. Yes. I should've mentioned that.

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -1370,7 +1370,6 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	icon_state = "bipod"
 	slot = ATTACHMENT_SLOT_UNDER
 	size_mod = 2
-	melee_mod = -10
```
[the remark this answers is not in the document]
Comment (Mrrpip) [reply code/modules/projectiles/gun_attachables.dm:1373]: ...How even...?

**Comment by Mrrpip** [comment]
...Well then.

**Review by ivanmixo — APPROVED** [review]
Specify in the description/name that it's meant for autoshotty otherwise people will be confused why it doesn't fit in the normal shotguns (this was meant to be change request)

**Comment by Mrrpip** [comment]
Done, it's a bit bland pointing it out, but it'll make sure that doesn't happen.

**Comment by Arturlang** [comment]
Doesn't it already have it in the form of shotgun ammo boxes, how does it compare to those, or is it just a plain better version powercreep?

**Comment by Xsdew** [comment]
> Don't shells already have the semi-portable ammo boxes, how does it compare to those, or is it just a plain better version powercreep?

Its for the magazine fed shotgun. Which doesnt have 25 round boxes.

**Comment by Mrrpip** [comment]
Mhm-hm! As Xsdew mentioned, it's not useable with the usual 12 Gauges.

A bit of helpful trivia though, a good amount of other (Small and pocketable/baggable) loose ammo boxes carry anywhere from 3 (Most AR boxes.) to 5 mags worth (In the case of the TL-127).

This falls in the middle at 4 mags of 15.

Hope that helps!

**Pull request merged**